### ISSUE TITLE
Library source does not match the bytecode for class CronSchedule

### ISSUE THREAD

**Issue by virtual-machinist** [kagkarlsson/db-scheduler#120]
A nice little library I'm currently evaluating instead of the (in)famous Quartz.

One small issue though. When examining the code I get `Library source does not match the bytecode for class CronSchedule` warning in IntelliJ IDEA. That's because the sources JAR contains
```
import com.cronutils.model.Cron;
import com.cronutils.model.CronType;
import com.cronutils.model.definition.CronDefinitionBuilder;
import com.cronutils.model.time.ExecutionTime;
import com.cronutils.parser.CronParser;
```
And the compiled class file has `com.cronutils` shaded under `com.github.kagkarlsson.shaded.cronutils` package. This is a bit of annoying since I cannot navigate the code further and instead have to browse `com.cronutils:cron-utils` project in a separate window.

Is there any way to provide a matching sources JAR? I believe PgJDBC project did that somehow for `com.ongres.scram` even before switching to Gradle.

**Comment by kagkarlsson** [comment]
I can see why that is annoying. Will see that I can do 👍

**Comment by kagkarlsson** [comment]
Hopefully this is fixed in v`8.1`, otherwise let me know
